src/extensions/code_quote.py

```diff
@@ -0,0 +1,81 @@
+import logging
```

Comment:
i assuming the rest of these were just moved from the old extensions folder (that was outside)

Reply:
Yes. This is a file move, relative to the last commit on main.